apps/etterlatte-hendelser-pdl/.nais/dev.yaml

```diff
@@ -21,8 +21,6 @@ spec:
   azure:
```

Comment:
Hvorfor revertes ikke mergen og reopenerer med rebasing mot main? Nå blir historien feil, hva med andre som har tatt utgangspunkt i main som en annen historikk enn det her?

Reply:
Ah, sånn ja. Først ein revert og så ein ny PR ut frå samme grein, berre riktig rebasa. Jo, kan for så vidt det.

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/792